game/static/game/css/auth.css

```
@@ -1,4 +1,4 @@
-/* auth.css - Shared styles for Checkora Authentication Pages */
+/* auth.css - Shared styles for Checkora Authentication Pages*/
```

Stylelint (17.11.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)